mirdata/validate.py

```diff
@@ -48,7 +50,9 @@ def validate(local_path, checksum):
 
     """
     # validate that the file exists on disk
-    if not os.path.exists(local_path):
+    try:
+        open(local_path)
```

Comment:
shouldn't this be a with open with a pass? Otherwise I belive you still need to cache the variable and close it